tools/binlog/deploy.md

```diff
@@ -0,0 +1,488 @@
+
+        默认 Pump 保留 5 天数据，如需修改可修改 `tidb-ansible/conf/pump.yml` 文件中 `gc` 变量值，并取消注释，如修改为 7。
+
+        ```yaml
+        global:
+          # an integer value to control the expiry date of the binlog data, which indicates for how long (in days) the binlog data would be stored
+          # must be bigger than 0
+          gc: 7
+        ```
+
+        请确保部署目录有足够空间存储 binlog，详见：[部署目录调整](../../op-guide/ansible-deployment.md#部署目录调整)，也可为 Pump 设置单独的部署目录。
+
+        ```ini
+        ## Binlog Part
+        [pump_servers]
+        pump1 ansible_host=172.16.10.72 deploy_dir=/data1/pump
+        pump2 ansible_host=172.16.10.73 deploy_dir=/data1/pump
+        pump3 ansible_host=172.16.10.74 deploy_dir=/data1/pump
+        ```
+
+2. 部署并启动含 Pump 组件的 TiDB 集群
+
+    参照上文配置完 `inventory.ini` 文件后，从以下两种方式中选择一种进行部署。
+    
+    **方式一**：在已有的 TiDB 集群上增加 Pump 组件，需按以下步骤逐步进行。
+    
+    1. 部署 pump_servers 和 node_exporters
+    
+        ```
+        ansible-playbook deploy.yml -l ${pump1_ip}, ${pump2_ip}, [${alias1_name}, ${alias2_name}]
+        ```
+        
+    2. 启动 pump_servers
+    
+        ```
+        ansible-playbook start.yml --tags=pump
+        ```
+        
+    3. 更新并重启 tidb_servers
+    
+        ```
+        ansible-playbook rolling_update.yml --tags=tidb
+        ```
+        
+    4. 更新监控信息
+    
+        ```
+        ansible-playbook rolling_update_monitor.yml --tags=prometheus
+        ```
+
+    **方式二**：从零开始部署含 Pump 组件的 TiDB 集群
+    
+    使用 Ansible 部署 TiDB 集群，方法参考 [TiDB Ansible 部署方案](../../op-guide/ansible-deployment.md)。
+
+3. 查看 Pump 服务状态
+
+    使用 binlogctl 查看 Pump 服务状态，pd-urls 参数请替换为集群 PD 地址，结果 State 为 online 表示 Pump 启动成功。
+
+    ```bash
+    $ cd /home/tidb/tidb-ansible
+    $ resources/bin/binlogctl -pd-urls=http://172.16.10.72:2379 -cmd pumps
+
+    INFO[0000] pump: {NodeID: ip-172-16-10-72:8250, Addr: 172.16.10.72:8250, State: online, MaxCommitTS: 403051525690884099, UpdateTime: 2018-12-25 14:23:37 +0800 CST}
+    INFO[0000] pump: {NodeID: ip-172-16-10-73:8250, Addr: 172.16.10.73:8250, State: online, MaxCommitTS: 403051525703991299, UpdateTime: 2018-12-25 14:23:36 +0800 CST}
+    INFO[0000] pump: {NodeID: ip-172-16-10-74:8250, Addr: 172.16.10.74:8250, State: online, MaxCommitTS: 403051525717360643, UpdateTime: 2018-12-25 14:23:35 +0800 CST}
+    ```
+### 第 3 步：部署 Drainer
+<a href="#第 3 步：部署 Drainer"></a>
+
+
+1. 获取 initial_commit_ts
+
+    使用 binlogctl 工具生成 Drainer 初次启动所需的 tso 信息，命令：
+
+    ```bash
+    $ cd /home/tidb/tidb-ansible
+    $ resources/bin/binlogctl -pd-urls=http://127.0.0.1:2379 -cmd generate_meta
+    INFO[0000] [pd] create pd client with endpoints [http://192.168.199.118:32379]
+    INFO[0000] [pd] leader switches to: http://192.168.199.118:32379, previous:
+    INFO[0000] [pd] init cluster id 6569368151110378289
+    2018/06/21 11:24:47 meta.go:117: [info] meta: &{CommitTS:400962745252184065}
+    ```
+
+    该命令会输出 `meta: &{CommitTS:400962745252184065}`，CommitTS 的值作为 Drainer 初次启动使用的 `initial-commit-ts` 参数的值。
+
+2. 全量数据的备份与恢复
```

Comment:
Is this step necessary?

Reply:
No need, put it here will be confusing to users. I will remove it.

Reply:
Updated in https://github.com/pingcap/docs-cn/pull/1264/commits/4359073d86499983ffe7cf3c76ec84dd747fd565